monkey/infection_monkey/exploit/vsftpd.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from logging import getLogger
+
+import time
+
+import paramiko
+import StringIO
+import socket 
+import sys 
+import time
+
+import infection_monkey.monkeyfs as monkeyfs
```

Comment:
A lot of unused import statements

Reply:
Removed sys and  monkeyfs as they aren't being used anymore